java/arcs/core/host/AbstractArcHost.kt

```diff
@@ -99,10 +119,33 @@ abstract class AbstractArcHost(
     // TODO: add lifecycle API for ArcHosts shutting down to cancel running coroutines
     private val scope = CoroutineScope(coroutineContext)
 
+    /**
+     * Supports asynchronous [ArcHostContext] serializations in observed order.
+     *
+     * TODO:
+     * make the channel per-Arc instead of per-Host for better serialization
+     * performance under multiple running and to-be-run Arcs.
+     */
+    private val contextSerializationChannel: Channel<suspend () -> Unit> =
+        Channel(Channel.UNLIMITED)
+    private val contextSerializationTasks =
```

Comment:
`contextSerializationJob` may be a better name, since the returned type is `Job`.

Reply:
ok! will rename it, thanks